src/components/LimitedAccessBanner/LimitedAccessBanner.css

```diff
@@ -0,0 +1,31 @@
+@import '../../marketplace.css';
+
+.root {
+  background-color: #df492a;
```

Comment:
I'm not sure if this color should come from `marketplace.css` so that it would be easier to change to be in line with other marketplace colors. On the other hand, I guess the notification should only be visible to the operator so not sure if this is really needed.

Reply:
Good reminder regarding the colors. I think it makes sense to use the `--marketplaceColor` in the background. The grey button however, is a bit hard to make work with the colors from marketplace.css I think with it we can stick to the custom colors.

Reply:
On second thoughts, if the banner bg color is the marketplace color it might go unnoticed. Therefore, it's good that it's a notable color and differs from the marketplace main color. I'll keep the custom colors as this is meant to communicate a special behaviour.